dev/aarch64_opt/src/README.md

```diff
@@ -0,0 +1,62 @@
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# mlkem-native AArch64 back-end SLOTHY-optimized code
+
+This directory contains the AArch64 back-end _after_ it has
+been optimized by SLOTHY.
+
+## Running SLOTHY from the Makefile
+
+If the "clean" back-end sources in `../../aarch64_clean/src/*.S` change,
```

Comment:
Use a markdown link please to have it checked by CI. E.g. 
```suggestion
If the "clean" back-end sources in [`../../aarch64_clean/src/*.S`](../../aarch64_clean/src/) change,
```

Reply:
Done